students/grzegorz_bajorski/lesson_04_unit_testing/fibonacci_numbers_tests.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from fibonacci_numbers import fib
+
+
+class TestFib(unittest.TestCase):
+
+    def test_0(self):
+        self.assertRaises(TypeError, fib, 'aoeu')
+
+    def test_1(self):
+        self.assertRaises(fib(8), 4)
```

Comment:
This test is failing due to:
- wrong assertion type, where you are comparing two values use assertEqual()
- fib(8) is returning the different value than 4